lib/components/credits.dart

```diff
@@ -0,0 +1,330 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+  
+
+class Credits extends StatelessWidget {
+  @override
+Widget build(BuildContext context){  
+   return SingleChildScrollView(
+   child:Container(
+     color: Colors.white,
+     child:Column( 
+    children: <Widget>[
+        Row(
+        children: <Widget>[ 
+     GestureDetector(
+       onTap: () {  
+          Navigator.of(context).pop(); 
+        },  
+       child:Container(
+        child: Icon(Icons.arrow_back_ios),
+      ),
+      ),
+      Container(
+ 
+        height: 100,
+    margin: const EdgeInsets.only(left:69.0),
+    alignment: Alignment.center, 
+    child: Text(
+      "Developers",
+      style:TextStyle(
+        fontSize: 25,
+        fontWeight: FontWeight.w100, 
+        fontFamily: 'B612',
+        color: Colors.black
+        ),
+      ),  
+    ),
+        ]
+      ),
+    Center( 
+        child:Container(
+          margin: const EdgeInsets.all(2.0),
+          decoration:BoxDecoration(
+            borderRadius: BorderRadius.circular(10.00),
+            border: Border.all( 
+              color: Colors.blueAccent
+                )
+              ),     
+            child:Container(
+              height: 100.0,
+              width: 100.0,
+              child: Material(
+              borderRadius: BorderRadius.circular(10.0),
+              shadowColor: Color(0xFFb3e5fc),
+              //color: Color(0xFF039be5),
+              elevation: 7.0,
+                  child:GestureDetector(
+                    onTap: (){
+                      launch('https://zairza.in/');
+                          },
+                    child: Container(                 
+                    //margin:EdgeInsets.fromLTRB(1.0,10.0,150.0,10.0) ,
+                    decoration: BoxDecoration(
+                      image: DecorationImage(
+                        image: AssetImage(
+                        'assets/zairza_logo.jpg'),
+                        fit: BoxFit.fill
+                            ),
```

Comment:
<img width="368" alt="Screenshot 2020-07-25 at 3 55 03 PM" src="https://user-images.githubusercontent.com/26324376/88454853-e40e2980-ce8f-11ea-97ce-37d70d127cef.png">

The design does not look good IMO, to make a rather consistent design make this the exact size of the "options" dialog and scroll for more details. Add proper spaces and padding for the content/make dialog bigger.